Tests/SourceKittenFrameworkTests/Fixtures/Realm@swift-3.2.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "Realm\/RLMArray.h" : {
+    "Realm\/Realm\/RLMArray.h" : {
```

Comment:
This file should be unchanged.

Reply:
I moved the realm header fixtures down a level to add the xcodeproject files somewhere that looked neat (to me!).

Reply:
ok, that's fine